src/glyphs/font-awesome/mapping

```diff
@@ -478,6 +478,7 @@ F69D EEE4 brands/the-red-yeti.svg the_red_yeti
 F6A0 EEE5 solid/scroll-torah.svg scroll_torah
 F6A1 EEE6 solid/torii-gate.svg torii_gate
 F6A7 EEE7 solid/vihara.svg vihara
+F6A8 F6A8 solid/volume.svg volume
```

Comment:
The second column is the Nerd Fonts codepoint, i.e. `torii_gate` is on `U+EEE6`, `vihara` is on `U+EEE7`,  and in this PR the new `volume` is put on `U+F6A8`.

That is not even in the patching range for Font Awesome, which is `ed00` - `efce` and `f000` - `f2ff`.

https://github.com/ryanoasis/nerd-fonts/wiki/Glyph-Sets-and-Code-Points#overview

See
* https://github.com/ryanoasis/nerd-fonts/issues/365

why we can not use `U+F6A8`.